src/app.jsx

```diff
@@ -293,3 +344,12 @@ export const ViewSelector = ({ isGrid, setIsGrid, sortBy, setSortBy }) => {
         </Select>
     );
 };
+
+const getFormattedDateTime = (time) => {
```

Comment:
Couldn't that use pkg/lib/timeformat.js, dateTimeSeconds()? distanceToNow() may also be interesting (but perhaps as additional output, not the only one)